simms/skymodel/source_factory.py

```diff
@@ -280,7 +292,15 @@ def is_point(self):
 
     @property
     def is_transient(self):
-        return self.transient_start not in [None, "null"]
+        if any(
```

Comment:
This is incomplete. Since all of these are required for transient source, there should be an inner loop checking for unset transient parameters. If any are found, you should raise an exception (a `SkymodelError`) telling the user which parameters are missing.